jwst/background/background_sub.py

```diff
@@ -97,32 +210,37 @@ def average_background(bkg_list, sigma, maxiters):
     # Loop over the images to be used as background
     for i, bkg_file in enumerate(bkg_list):
         log.info(f'Accumulate bkg from {bkg_file}')
-        bkg_model = datamodels.open(bkg_file)
+
+        bkg_array = ImageSubsetArray(bkg_file)
+        if not bkg_array.overlaps(im_array):
```

Comment:
I'm thinking it might be useful to report non-overlap cases in the log, at least at the debug level, if not info. Just so user has an understanding of what's being included in the average bkg.